sql/upgrade/sl2.8-1.5.sql

```diff
@@ -724,6 +724,7 @@ SELECT setval('id', max(id)) FROM transactions;
  SELECT setval('cr_report_line_id_seq', max(id)) FROM cr_report_line;
  SELECT setval('business_unit_id_seq', max(id)) FROM business_unit;
 
+INSERT INTO defaults(setting_key,value) VALUES('default_country', :default_country);
```

Comment:
Isn't this unrelated to the change at hand?

Reply:
On the contrary, we need to stash this value if we want to be able to use it as default selection for new users, based on the hypothesis that 99% of the time, the users will be in the same country as the company. So it can serve as a default for the initial user creation.